### ISSUE TITLE
Unable to install in symfony3

### ISSUE THREAD

**Issue by vishalmelmatti** [FriendsOfSymfony/FOSMessageBundle#274]
Hi, 

I am trying to install it using composer in symfony3, but i am getting these erros.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - friendsofsymfony/message-bundle v1.2.2 requires symfony/framework-bundle ~2.1 -> no matching package found.
    - friendsofsymfony/message-bundle v1.2.1 requires symfony/framework-bundle ~2.1 -> no matching package found.
    - friendsofsymfony/message-bundle v1.2.0 requires symfony/framework-bundle >=2.1,<2.3-dev -> no matching package found.
    - Installation request for friendsofsymfony/message-bundle ^1.2 -> satisfiable by friendsofsymfony/message-bundle[v1.2.0, v1.2.1, v1.2.2].

```

**Comment by tgalopin** [comment]
That's a good point. I'll try to fix that.

**Comment by tgalopin** [comment]
I'm working on Symfony 3 support here: https://github.com/FriendsOfSymfony/FOSMessageBundle/pull/275

It's not an easy task, especially on Forms.

**Comment by tgalopin** [comment]
Should be soon merged in https://github.com/FriendsOfSymfony/FOSMessageBundle/pull/289.

**Comment by tgalopin** [comment]
Merged in #289.
